src/toil/common.py

```diff
@@ -524,3 +530,36 @@ def parseSetEnv(l):
             raise ValueError('Empty name')
         d[k] = v
     return d
+
+
+def getToilWorkflowDir(workflowID, configWorkDir=None):
+    """
+    Returns a path to the directory where worker directories and the cache will be located for this
+    workflow.  This is particularly useful if the user hasn't provided a value for --workDir (passed
```

Comment:
cut everything after the first sentence. A docstring should describe what a function does, not how.